src/Editor/index.js

```diff
@@ -517,12 +530,15 @@ export class Editor extends React.Component {
                                 name={'message'}
                                 value={state.inputText}
                                 onBlur={props.toggleEditor}
+                                onKeyPress={this._onKeyPress}
                                 onChangeText={this.onChange}
                                 selection={this.state.selection}
                                 selectionColor="rgba(0, 0, 0, 0.5)"
                                 onSelectionChange={this.handleSelectionChange}
                                 onContentSizeChange={this.onContentSizeChange}
                                 scrollEnabled={false}
+                                keyboardType={'email-address'}
```

Comment:
I'm not sure it's a good idea to use email-address, as that would make typing commas (`,`), quite hard.

See https://lefkowitz.me/visual-guide-to-react-native-textinput-keyboardtype-options/.

Also, in this case, you can set the prop as just `keyboardType="email-address"`

Reply:
I agree, its mostly a temporal solution while we don't have a design for a `@` button, but I can remove it if you like.